GrotonChat/GrotonChatCensor.py

```diff
@@ -0,0 +1,93 @@
+import requests
+import json
+
+SENTIM = "https://sentim-api.onrender.com/api/v1/"
+
+# TODO: Add elements to each of the lists below.  Then
+# good_things list should contain strings relevant to Groton.
+# The bad_things list should contain strings relevant to
+# St. Marks (or some other objectively bad place).
+good_things = ["groton", "schoolhouse", "zebra" "tight-knit community"]
+bad_things = ["st.marks", "lion", "southborough" ]
+
+#TODO:  Initialize the dictionary below by adding in the special
+# "<ADMIN>" key.  Set it to an initial value of 0.
+scores = {"<ADMIN>":0}
+
+def check_needs_score(good_things : list, \
+                      bad_things : list , \
+                      message : str) -> (bool, str):
+    lowercase_message = message.lower()
+    for word in lowercase_message.split():
+        print(word)
+        for good_thing in good_things:
+            if good_thing.lower() == word:
+                return True, "positive"
+        for bad_thing in bad_things:
+            if bad_thing.lower() == word:
+                return True, "negative"
+    return False, "positive" 
+
+def get_adjustment(text:str, desired_type:str) -> float:
+    resp = requests.post(SENTIM, json={"text": text})
+    j = resp.json()
+    polarity = j["result"]["polarity"]
+    analysis_type = j["result"]["type"]
+    if analysis_type == desired_type:
+        adjustment = abs(polarity)
+    else:
+        adjustment = -abs(polarity)
+    
+    return adjustment
+
+def score(message:str) -> (str, float, float, str):
+    user, text = message.split('\t')
+
+    # TODO: Update the line below to call the check_needs_score
+    # function with the appropiate parameters to
+    # determine if the message needs to be scored.
+    needs_score, desired_type = check_needs_score(good_things, bad_things, text)
+    print(check_needs_score(good_things, bad_things, text))
+    adjustment = 0
+    polarity = 0
+
+    if needs_score:
+        # Call the get_adjustment function to determine the score adjustment
+        adjustment = get_adjustment(text, desired_type)
+
+    # Update the user's score based on the adjustment calculated above
+    scores[user] = scores.get(user, 0) + adjustment
+
+    # Check conditions for banning users or censoring messages
+    if scores.get(user, 0) < -2:
```

Comment:
You know for sure from line 59 that `scores[user]` exists, so you could write the following instead:

```suggestion
    if scores[user] < -2:
```